editor/osm_editor.cpp

```diff
@@ -935,7 +936,9 @@ bool Editor::FillFeatureInfo(FeatureStatus status, XMLFeature const & xml, Featu
   EditJournal journal = xml.GetEditJournal();
 
   // Do not load Legacy Objects form Journal
-  bool loadFromJournal = journal.GetJournalHistory().front().journalEntryType != JournalEntryType::LegacyObject;
+  auto const & journalHistory = journal.GetJournalHistory();
+  bool loadFromJournal = journalHistory.empty() || journalHistory.front().journalEntryType != JournalEntryType::LegacyObject;
```

Comment:
Tale out into a separate function to avoid copy-paste-check ?

Reply:
It's not really worth it because I want to remove the decision logic for old/new editor in a few months anyway. This code is only needed for the transition period.